Test/comp/IKnowThisMuchIs.dfy.expect

```diff
@@ -8,5 +8,6 @@ true
 
 Dafny program verifier finished with 0 verified, 0 errors
 true
+
```

Comment:
Why is there a newline here?

Reply:
Python output is like that same as java above in the same file., and I was getting some problems keeping a newLine at the end of file due to formatters so I have moved the line for ```py``` above ```js``` and added a blank line after the current output and added the the same output again as the output is same.